src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * @param string $sku
+     * @param int    $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
```

Comment:
Anyway I would like to keep domain VO construction at application layer, if you need to pass them from UI controllers are growing quite fast